tests/compile.py

```diff
@@ -0,0 +1,8 @@
+from os.path import join, dirname
+import os
+import subprocess
+
+pth = join(dirname(dirname(__file__)), 'aequilibrae', 'paths')
+
+os.chdir(pth)
+subprocess.Popen("python setup_Assignment.py build_ext --inplace", shell=True, stdout=subprocess.PIPE).stdout.read()
```

Comment:
Is it worth using a substitution here

```suggestion
subprocess.Popen(f"{sys.executable} setup_Assignment.py build_ext --inplace", shell=True, stdout=subprocess.PIPE).stdout.read()
```

to make sure the right python is called.